msa/js-executor/api/jsInvokeMessageProcessor.js

```diff
@@ -115,15 +116,20 @@ JsInvokeMessageProcessor.prototype.processCompileRequest = function(requestId, r
     );
 }
 
-JsInvokeMessageProcessor.prototype.processInvokeRequest = function(requestId, responseTopic, headers, invokeRequest) {
+JsInvokeMessageProcessor.prototype.processInvokeRequest = function (requestId, responseTopic, headers, invokeRequest) {
     var scriptId = getScriptId(invokeRequest);
     logger.debug('[%s] Processing invoke request, scriptId: [%s]', requestId, scriptId);
     this.executedScriptsCounter++;
-    if ( this.executedScriptsCounter >= scriptBodyTraceFrequency ) {
-        this.executedScriptsCounter = 0;
-        if (logger.levels[logger.level] >= logger.levels['debug']) {
-            logger.debug('[%s] Executing script body: [%s]', scriptId, invokeRequest.scriptBody);
-        }
+    if (this.executedScriptsCounter % statFrequency == 0) {
+        var nowMs = performance.now();
+        var msSinceLastStat = nowMs - this.lastStatTime;
```

Comment:
please put Math.max(nowMs - this.lastStatTime, 1) just in case..

Reply:
fixed with zero check